assets/shaders/texture_binding_array.wgsl

```diff
@@ -0,0 +1,19 @@
+@group(1) @binding(0)
+var textures: binding_array<texture_2d<f32>>;
+@group(1) @binding(1)
+var samplers: binding_array<sampler>;
+
+@fragment
+fn fragment(
+    #import bevy_pbr::mesh_vertex_output
+) -> @location(0) vec4<f32> {
+#ifdef VERTEX_UVS
```

Comment:
For the sake of keeping the example simple I'd just remove this ifdef and assume it's always present